spec/02-integration/01-cmd/02-start_stop_spec.lua

```diff
@@ -29,7 +31,9 @@ describe("kong start/stop", function()
   end)
   it("start/stop custom Kong conf/prefix", function()
     assert(helpers.kong_exec("start --conf "..helpers.test_conf_path))
+    assert.True(#pl_dir.getfiles(pl_path.join(helpers.test_conf.prefix, "pids")) > 0)
     assert(helpers.kong_exec("stop --prefix "..helpers.test_conf.prefix))
+    assert.True(#pl_dir.getfiles(pl_path.join(helpers.test_conf.prefix, "pids")) == 0)
```

Comment:
use `helpers.dir.getfiles` and `helpers.path.join` instead of requiring `pl.dir` and `pl.path`.